backend/openapi2-hermes-v2.yaml

```diff
@@ -807,44 +834,48 @@ paths:
           description: Internal server error, please contact support
           schema:
             $ref: '#/definitions/ErrorDTO'
-  /timetables/zips:
+  /publications/{id}:
     get:
       tags:
-        - zips
-      operationId: "generateSchedulesZip"
-      summary: Creates zip file with pdf schedules according to given parameters
-      produces:
-        - application/zip
+        - publications
+      operationId: "getPublication"
+      summary: Gets a publication by id
       parameters:
-        - name: lineId
-          in: query
-          description: Id of the line for whose station pdf is generated
-          required: false
-          type: string
-        - name: stationId
-          in: query
-          description: Id of the station for which pdf is generated
-          required: false
-          type: string
-        - name: scheduleIds
-          in: query
-          description: List of ids of schedules to be included in pdf
+        - name: id
+          in: path
+          description: ID of a Publication to fetch
           required: true
-          type: array
-          minItems: 1
-          items:
-            type: string
+          type: string
       responses:
         200:
-          description: Zip file with pdf schedules
+          description: Publication fetched
           schema:
-            type: file
+            $ref: '#/definitions/PublicationDTO'
         404:
-          description: Line, station or one of schedules is not found
-        400:
-          description: Problem with the request, such as a missing, invalid or type mismatched parameter
+          description: Publication not found
+        500:
+          description: Internal server error, please contact support
           schema:
             $ref: '#/definitions/ErrorDTO'
+  /publications/{id}/files:
```

Comment:
Can we use `{id}/contents`? `Files` is a bit confusing since it can be file or zip

Reply:
Sure